src/Handler/Mapping/ClassName/Suffix.php

```diff
@@ -4,11 +4,18 @@
 
 namespace League\Tactician\Handler\Mapping\ClassName;
 
-class Suffix implements ClassNameInflector
+final class Suffix implements ClassNameInflector
 {
+    private const DEFAULT = 'Handler';
+
     /** @var string */
     private $suffix;
 
+    public static function handler(): self
+    {
+        return new self(self::DEFAULT);
+    }
+
```

Comment:
Dig the final and while I like the idea, I'd like to avoid supporting too many direct defaults in T2, so would prefer to drop this.